pages/landingpages/introducing-dataops-into-your-data-management-discipline-gartner/index.html

```diff
@@ -68,7 +68,7 @@ <h5>COMPLIMENTARY REPORT</h5>
                         <img src="{{ BASE_PATH }}/v3/assets/img/global/tools/gartner-logo.svg" alt="Gartner">
                         <h3>Introducing DataOps into your data management discipline</h3>
                     </section>
-                    <form class="box-shadow" id="main-form" data-pardotUrl="https://go.snowplowanalytics.com/l/571483/2020-06-18/3s7pqb8" data-gtmEventName="webinar-3-success" novalidate>
+                    <form class="box-shadow" id="main-form" data-pardotUrl="https://go.snowplowanalytics.com/l/571483/2020-06-18/3s7pqb8" data-gtmEventName="gartner-dataops-success" novalidate>
```

Comment:
Are these other files intended to be published?

Reply:
Not sure that happened when I changed the base branch but I am sure Daniel will be able to sort it out!